MAINTAINERS.md

```diff
@@ -0,0 +1,51 @@
+# Helm Maintainers
+
+This document explains the leadership structure of the Kubernetes Helm project, and list the current project maintainers.
+
+The Helm project welcomes contributions from any interested parties. This document explains the role of what we call "Core Maintainers". Core Maintainers play the special role of curating the project.
+
+## What is a Core Maintainer?
+
+(Unabashedly stolen from the [Docker](https://github.com/docker/docker/blob/master/MAINTAINERS) project)
+
+There are different types of maintainers, with different responsibilities, but
+all maintainers have 3 things in common:
+
+1. They share responsibility in the project's success.
+2. They have made a long-term, recurring time investment to improve the project.
+3. They spend that time doing whatever needs to be done, not necessarily what
+is the most interesting or fun.
+
+## Helm Core Maintainers
+
+The duties of a core maintainer include:
+
+* Classify and respond to GitHub issues and review pull requests
+* Perform code reviews (see below)
+* Shape the Helm roadmap and lead efforts to accomplish roadmap milestones
+* Participate actively in feature development and bug fixing
+* Answer questions and help users
+
+The current core maintainers of Helm (in alphabetical order):
+
+* Ville Aikas - [@vaikas-google](https://github.com/vaikas-google)
+* Matt Butcher - [@technosophos](https://github.com/technosophos)
+* Jack Greenfield - [@jackgr](https://github.com/jackgr)
+* Brendan Melville - [@bmelville](https://github.com/bmelville)
+* Michelle Noorali - [@michelleN](https://github.com/michelleN)
+* Adam Reese - [@adamreese](https://github.com/adamreese)
+
+### Code Reviews
+
+Before being merged, every pull request must have at least two LGTMs ("Looks Good To Me"), and at least one of these must come from one of the core maintainers listed above.
```

Comment:
So, we've been using 1 LGTM due to the small size of our team. Also, for minor changes, it seems like overkill to require 2 LGTMs. Keep in mind that Kubernetes has about 20 active committers.

In the trade-off between big up front design and rapid iteration, I'm always a fan of the latter. For that reason, keeping the velocity up matters more to me than being 100% certain of what we're committing. Experience suggests that 100% certainty is a myth, anyway.

Worst case, if a committer pushes something that we really don't want, and the 1 reviewer doesn't catch it, we can always back it out. That has happened many times in k8s land, despite their 2 LGTMs. It's a complex product with many opportunities for side effects. Ours is simpler, so I don't think we'll see that need to back out commits as often as they do.